client/src/components/Home/Home.css

```diff
@@ -0,0 +1,4 @@
+.lander-img{
+width:auto;
+height: auto;
```

Comment:
Do take care of the indentation.

Reply:
As I postscript 😅 the cover image was for illustration purpose so I didn't decor that code as I haven't started working on that.